pkg/storaged/client.js

```diff
@@ -970,11 +988,11 @@ function stratis3_start() {
                     return client.stratis_manager.StartPool(uuid, [!!unlock_method, unlock_method || ""]);
                 };
 
-                client.stratis_create_pool = (name, devs, key_desc) => {
+                client.stratis_create_pool = (name, devs, key_desc, clevis_info) => {
                     return client.stratis_manager.CreatePool(name, [false, 0],
                                                              devs,
                                                              key_desc ? [true, key_desc] : [false, ""],
-                                                             [false, ["", ""]]);
+                                                             clevis_info ? [true, clevis_info] : [false, ["", ""]]);
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18818-20230606-121339-eda190c6-fedora-38-devel//Coverage/lcov/github-pr.diff.gcov.html)